### DISCUSSION TITLE
[Meetup] Zapisy na sesje na 46 spotkanie Microsoft Azure User Group Poland

### DISCUSSION THREAD

**Discussion by aryll** [AzureCommunityPL/group#251] [Wydarzenia]
🌻 Pierwsze skowronki wróciły, przebiśniegi pojawiły się na wzgórzach, a słońce coraz odważniej wychodzi zza chmur. To oczywisty znak, że **następny meetup grupy** przed nami!

📅 Nadal czekamy na spotkania stacjonarne, ale w międzyczasie kolejne spotkanie online odbędzie się już **14 kwietnia 2021 (środa) o godzinie 18:00**!

✔️ Chcesz wystąpić na spotkaniu? Wciąż mamy wolne sloty na sesję, więc jeśli chciałbyś /chciałabyś wystąpić, to kliknij link poniżej i zgłoś swoją sesję:

### [Link do propozycji sesji](https://azurecommunitypl.typeform.com/to/yF2MQWly)

**Comment by mgrabarz** [comment]
Super wiadomość, przy czym #45 był już w grudniu :)

Jeśli potrzebujesz pomocy, daj znać!

**Reply by aryll** [reply]
zmienione
